@types/modules/mewm/types.d.ts

```diff
@@ -0,0 +1,18 @@
+declare interface MEWMEventTypeUpdateRequest {
+  eventTypeName?: string;
+  eventTypeDescription?: string;
```

Comment:
Can we add the `permissions` object here?
```suggestion
  eventTypeDescription?: string;
  permissions?: UASPermissionSwitchConfig<boolean>;
```